assets/app/styles/_core.less

```diff
@@ -203,6 +203,19 @@ html, body {
   }
 }
 
+.nav-tabs {
+  font-size: @font-size-base;
+  a {
+    // ui.bootstrap.tabs doesn't set the href attr, so the cursor style is
```

Comment:
related, I assume there is some way on a tabset tab to specify if it is the active tab